database/seeding.js

```diff
@@ -0,0 +1,17 @@
+const data = require('./data.js');
+const mongoose = require('mongoose');
+const Description = require('./database.js');
+
+
+var fakeData = data.data;
+
+Description.Description.count({}, function(err, count) {
```

Comment:
You can export Description in database.js like this `module.exports = Description;` to avoid having this `Description.Description...` or import it in this file with a relevant name like db so you can have `db.Description...`